css/src/components/buttons.scss

```diff
@@ -5,6 +5,7 @@
 	flex-wrap: wrap;
 	align-items: center;
 	justify-content: flex-start;
+	gap: 0.5rem;
```

Comment:
Should this be a `$layout-` or `$spacer-` variable?

Reply:
I was actually following the pattern used already in this file, I guess I can clean it up a little and align with how we do it everywhere else

Reply:
Either way makes sense to me; feel free to mark this as resolved.

Reply:
Updated